website/docs/docs/build/project-variables.md

```diff
@@ -110,7 +110,7 @@ The order of precedence for variable declaration is as follows (highest priority
 
 </VersionBlock>
 
-If dbt is unable to find a definition for a variable after checking these four places, then a compilation error will be raised.
+If dbt is unable to find a definition for a variable after checking these five places, then a compilation error will be raised.
```

Comment:
It looks like we still list 4 places for 1.5 (it changes when you select the version in the version picker), so suggesting we be more specific:
```suggestion
If dbt is unable to find a definition for a variable after checking all possible variable declaration places, then a compilation error will be raised.
```